packages/card/src/Card.ts

```diff
@@ -228,6 +234,11 @@ export class Card extends LikeAnchor(
             <sp-asset id="cover-photo" variant=${ifDefined(this.asset)}>
                 <slot name="cover-photo"></slot>
             </sp-asset>
+            ${this.variant !== 'quiet' && !this.horizontal
+                ? html`
+                      <sp-divider size="s"></sp-divider>
```

Comment:
From the design, I was under the impression that any image should have a divider underneath it. Are you saying here that the gallery variant should not?
![Screenshot 2023-06-27 at 9 34 48 AM](https://github.com/adobe/spectrum-web-components/assets/99203545/899726c5-fa92-4ba8-8e29-a1c86f0ff8a5)

Reply:
Our examples in Spectrum CSS didn't distinguish `coverPhoto` vs `preview` very well so I assumed they both needed it, but are there different use cases?

Reply:
I don't know how to answer that question. It is not present in your [PR preview](https://pr-1693--spectrum-css.netlify.app/card-gallery.html) for CSS, which is why I asked.

Reply:
Oh good callout. I must have just forgotten to add it to that example since it's separate from the others. According to the design, all cards except for quiet ones should have a divider underneath the image. I'll make that update to my PR to add it into that example.